index.js

```diff
@@ -20,7 +20,7 @@ const exec = (cmd, args) => {
 	}
 };
 
-module.exports = input => {
+module.exports = (input, options) => {
```

Comment:
Done. I spied on [pageres](https://github.com/sindresorhus/pageres/blob/master/lib/index.js#L14) for style, but I suppose that project is older.